components/ProfileEdit/ProfileEditHeader.tsx

```diff
@@ -1,7 +1,7 @@
 import { Text } from '@chakra-ui/react';
 
-import Header from '@common/Header';
-import GoBackIcon from '@common/Header/GoBackIcon';
+import Header from 'components/common/Header';
+import GoBackIcon from 'components/common/Header/GoBackIcon';
```

Comment:
이 부분 제가 제대로 확인 못했네요,, 죄송합니다😭
아래와 같이 변경하면 더 좋을 것 같습니다!

```suggestion
import { Header, GoBackIcon } from 'components/common';
```

Reply:
오 꼼꼼히 확인해주셔서 감사합니다!!